lib/payment/lib/providers/stripe-provider.ts

```
@@ -0,0 +1,549 @@
+      } else {
+        subscriptionParams.trial_period_days = trialPeriodDays;
+      }
+
+      const subscription = await this.stripe.subscriptions.create(subscriptionParams);
+
+      // Get the payment_intent_id if available
+      let paymentIntentId: string | undefined;
+      if (subscription.latest_invoice && typeof subscription.latest_invoice !== 'string') {
+        // Use a type assertion with additional check
+        const invoice = subscription.latest_invoice as any;
+        if (invoice && invoice.payment_intent && typeof invoice.payment_intent !== 'string') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)